### ISSUE TITLE
The state of Indic Languages in NLP

### ISSUE THREAD

**Issue by CalmDownKarm** [pydatadelhi/talks#100]
- Abstract (2-3 lines)
The Hindi speaking internet is exploding, Hindi language newspapers absolutely outrank english language papers in terms of circulation. While there are a lot of players in the NLP space in India, those focusing on Indian languages are few and far between.  Let's talk about the situation, the environment, the tooling, the language models, the problems

- Brief Description and Contents to be covered
The biggest problems with the environment today is that the gap between state of the art results and tooling people use in production is extremely large. Organizations that do work on Indian languages in NLP almost never publish source code or open source tooling. I'm going to talk about these problems and more importantly what people can do to help bridge this gap. 

- Pre-requisites for the talk
Basic fundamentals of NLP are nice, but not absolutely necessary

- Time required for the talk
30 - 45 Minutes

- Link to slides
TBD

- Will you be doing hands-on demo as well?
No

- Link to ipython notebook (if any)

- About yourself
I'm a developer at gramener and a research assistant at IIITDelhi working on deep learning based approaches to NLP

- Are you comfortable if the talk is recorded and uploaded to PyData Delhi's YouTube channel ?
Sure

- Any query ?

**Comment by shagunsodhani** [comment]
Hey @CalmDownKarm Thank you for proposing the talk. It is an interesting and extremely relevant topic. I am looking forward to your slides.

**Comment by sanketverma1704** [comment]
Hi, @CalmDownKarm. Can you update the slides, please? And would you like to deliver this in our upcoming meetup?

**Comment by CalmDownKarm** [comment]
Yes to deliver this on the 15th, but earliest I can do slides would be this weekend

**Comment by sanketverma1704** [comment]
Yeah, sure. That works!

**Comment by CalmDownKarm** [comment]
https://docs.google.com/presentation/d/1moSbzjQYmBIOs7SUeb1nuEgUr8K_hBiu1IhaNG2Lg6U/edit?usp=sharing Here's an early draft of the slides.

**Comment by sanketverma1704** [comment]
@shagunsodhani please have a look.

**Comment by shagunsodhani** [comment]
Hey @CalmDownKarm The slides look good but I am worried that they would not be sufficient for a 3-45 min talk. If you are planning to talk about more things (or in more detail than what is appears on the slides right now), I would recommend that you add them to the slide for the benefit of the audience.

Best of luck for the talk!

**Comment by sanketverma1704** [comment]
Thanks for the talk, closing this now.
